packages/block-library/src/social-links/block.json

```diff
@@ -56,6 +56,14 @@
 				"type": "flex"
 			}
 		},
+		"color": {
+			"background": true,
+			"gradients": true,
+			"text": false,
+			"__experimentalDefaultControls": {
+				"background": false
```

Comment:
Not entirely sure why I had to explicitly define `text` and `__experimentalDefaultControls.background` as `false`, but a text control appeared and the background color control is default without them.